### PR TITLE
fix spelling

### PR DESCRIPTION
**Pull request by svenwltr**

### CODE REVIEW

**Comment by svenwltr** [comment]
Here you are @hendrik-rebuy ...

@rebuy-de/prp-partial-deployment-cleanup

File: verify.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func Verify(agent string, namespace consul.Key, path string) error {
 
 		for _, branch := range branches {
 			if !directories.Contains(branch.Name) {
-				log.Printf("%s/%s is defined on Consul, but don't exists on disk!", project, branch.Name)
+				log.Printf("%s/%s is defined on Consul, but doesn't exists on disk!", project, branch.Name)
```
Comment (hendrik-rebuy) [inline verify.go]: doesn't exist 😈

Comment (svenwltr) [inline verify.go]: jetzt zufrieden?

**Comment by hendrik-rebuy** [comment]
:+1:

**Comment by bjoernhaeuser** [comment]
👍

**Pull request merged**